PyInstaller/depend/imphook.py

```diff
@@ -592,8 +592,7 @@ def find_all_package_nodes(name):
                 # "imports_to_remove".
                 for dest in imports_to_remove & references:
                     self.module_graph.removeReference(src, dest)
-                    logger.info(
-                        "  Removing import of %s from module %s", dest, src)
+
```

Comment:
This one should be kept. Juste remove spaces at the beginning and replace "Removing" by "Excluding".

Reply:
Resolved with new commit